newm-server/src/main/kotlin/io/newm/server/features/user/repo/UserRepositoryImpl.kt

```diff
@@ -62,6 +62,9 @@ internal class UserRepositoryImpl(
                 this.websiteUrl = user.websiteUrl?.asValidUrl()
                 this.twitterUrl = user.twitterUrl?.asValidUrl()
                 this.instagramUrl = user.instagramUrl?.asValidUrl()
+                this.spotifyUrl = user.spotifyUrl?.asValidUrl()
```

Comment:
These are not necessarily URLs. We should only be storing the `#######` parts which is what we send to Eveara.

![image](https://github.com/projectNEWM/newm-server/assets/245918/3649b65b-4581-499d-a504-c70fba5318f2)

Reply:
@AndrewWestberg Let's discuss over slack